django_cyverse_auth/models.py

```diff
@@ -10,7 +10,6 @@
 from django.db import models
 from django.conf import settings
 from django.utils import timezone
-from .models import Token as AuthToken
```

Comment:
This imports the class Token as AuthToken from models, which just happens to be this file!

See line: